docs/src/content/docs/de/components/tabs.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Registerkarten
+description: Lerne, wie du in Starlight Oberflächen mit Registerkarten erstellst, um gleichwertige Informationen zu gruppieren.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+Um eine Oberfläche mit Registerkarten zu erstellen, verwende die Komponenten `<Tabs>` und `<TabItem>`.
+Registerkarten sind nützlich, um gleichwertige Informationen zu gruppieren, wenn ein Nutzer nur eine von mehreren Optionen sehen muss.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<Tabs slot="preview">
+	<TabItem label="Stars">Sirius, Wega, Betelgeuse</TabItem>
+	<TabItem label="Moons">Io, Europa, Ganymed</TabItem>
+</Tabs>
+
+</Preview>
+
+## Import
+
+```tsx
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+```
+
+## Verwendung
+
+Zeige eine Oberfläche mit Registerkarten an, indem du die Komponenten `<Tabs>` und `<TabItem>` benutzt.
+Jedes `<TabItem>` muss ein [`label`](#label) haben, um es den Benutzern anzuzeigen.
+
+<Preview>
+
+```mdx
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+<Tabs>
+	<TabItem label="Stars">Sirius, Wega, Betelgeuse</TabItem>
+	<TabItem label="Moons">Io, Europa, Ganymed</TabItem>
+</Tabs>
+```
+
+<Fragment slot="markdoc">
+
+```markdoc
+{% tabs %}
+{% tabitem label="Stars" %}
+Sirius, Wega, Betelgeuse
+{% /tabitem %}
+
+{% tabitem label="Moons" %}
+Io, Europa, Ganymed
+{% /tabitem %}
+{% /tabs %}
+```
+
+</Fragment>
+
+<Tabs slot="preview">
+	<TabItem label="Stars">Sirius, Wega, Betelgeuse</TabItem>
+	<TabItem label="Moons">Io, Europa, Ganymed</TabItem>
+</Tabs>
+
+</Preview>
+
+### Registerkarten synchronisieren
+
+Halte mehrere Registerkartengruppen synchron, indem du das Attribut [`syncKey`](#synckey) hinzufügst.
+
+Alle `<Tabs>` auf einer Seite mit demselben Wert für `syncKey` zeigen dasselbe aktive Label an.
+
+Auf diese Weise kann dein Leser eine Auswahl treffen (z. B. sein Betriebssystem oder den Paketmanager), die dann beim Navigieren durch die Seiten beibehalten wird.
+
```

Comment:
```suggestion
Auf diese Weise kann dein Leser eine Auswahl treffen (z. B. sein Betriebssystem oder den Paketmanager), die dann beim Navigieren durch die Seiten beibehalten wird.
```

To match the original version.

Reply:
I'm sorry, but I don't understand what I need to change? May you please add an additional explanation maybe?

Reply:
Sorry, not sure what I did with this suggestion, I made another one (it's to remove an extra line) 😄

Reply:
No problem, happens to everybody... Thank you for having such a good eye, though! 🎉